shell/package.json

```diff
@@ -103,7 +104,6 @@
     "lodash": "4.17.21",
     "marked": "4.0.17",
     "nodemon": "2.0.22",
-    "nuxt": "2.15.8",
```

Comment:
@codyrancher can you remember why this was originally left in following the nuxt removal?

Reply:
No, I think it was just an oversight.